src/templates/admin/review/home.html

```diff
@@ -45,17 +46,42 @@ <h2>Articles in Review</h2>
                         {% if journal_settings.general.enable_expanded_review_details %}
                         <td>
                             {% if article.reviewassignment_set.exists %}
-
                                 <table style="width:100%; min-width:230px;" class="table table-stack small no-pad no-bottom-margin">
-                                    {% for review in article.reviewassignment_set.all %}
-
+                                    {% for review in article.reviews_not_withdrawn %}
                                         <tr>
+                                            <td>{{ review.review_round.round_number }}</td>
                                             <td><a href="{% url 'review_view_review' article.pk review.pk %}">{{ review.reviewer.initials }}</a></td>
-                                            <td><span class="{{ review.status.span_class }}">{{ review.status.display|safe }}</span></td>
+                                            <td>
+                                                <span class="{{ review.status.span_class }}">
+                                                    {% if review.status.code == 'complete' %}
+                                                    <span data-tooltip class="top" tabindex="2" title="Recommendation: {{ review.get_decision_display }}">
+                                                    {% endif %}
+                                                    {{ review.status.display|safe }}
+                                                    {% if review.status.code == 'complete' %}
+                                                    </span>
+                                                    {% endif %}
+                                                </span>
+                                            </td>
                                             <td>{{ review.status.date }}</td>
                                             <td>{% if review.status.reminder %}<a href="{% url 'review_send_reminder' article.pk review.pk review.status.reminder %}">{% endif %}Due {{ review.date_due|date:"d-m" }}{% if review.status.reminder %}</a>{% endif %}</td>
                                         </tr>
                                     {% endfor %}
+                                    {% if article.active_revisions %}
```

Comment:
I think we should aslo considere completed revisions within this round too here.

Once revisions are completed, it won't be apparent to the editor that the article is ready to act on